src/components/organisms/body-card.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react"
+import { useScroll } from "../../hooks/use-scroll"
+import Actionables, { ActionType } from "../molecules/actionables"
+
+type BodyCardProps = {
+  title?: string
+  subtitle?: string
+  events?: {
+    label: boolean
+    onClick: (e: React.MouseEvent<HTMLButtonElement>) => void
+  }[]
+  actionables?: ActionType[]
+}
+
+const BodyCard: React.FC<BodyCardProps> = ({
+  title,
+  subtitle,
+  events,
+  actionables,
+  children,
+}) => {
+  const { isScrolled, scrollListener } = useScroll()
+  return (
+    <div className="rounded-rounded border bg-grey-0 border-grey-20 h-full overflow-hidden flex flex-col min-h-[500px] w-full medium:w-1/2 relative">
```

Comment:
Should have a smaller min-height. See currency settings [here](https://www.figma.com/file/dlraOnQvQnrHQL0A7Ysbrs/Webapp-2.0?node-id=74%3A87).

Reply:
Suggestion would be `400px`